react/components/AccordionFilterContainer.js

```diff
@@ -99,58 +105,66 @@ const AccordionFilterContainer = ({
         )}
       </div>
 
-      {tree.length > 0 && (
-        <AccordionFilterItem
-          title={CATEGORIES_TITLE}
-          open={departmentsOpen}
-          show={!openItem || departmentsOpen}
-          onOpen={handleOpen(CATEGORIES_TITLE)}
-        >
-          <div className={itemClassName}>
-            <DepartmentFilters
-              tree={tree}
-              isVisible={tree.length > 0}
-              onCategorySelect={onCategorySelect}
-              hideBorder
-            />
-          </div>
-        </AccordionFilterItem>
-      )}
-
-      {nonEmptyFilters.map(filter => {
-        const { type, title } = filter
-        const isOpen = openItem === filter.title
-
-        switch (type) {
-          case 'PriceRanges':
-            return (
-              <AccordionFilterPriceRange
-                title={filter.title}
-                facets={filter.facets}
-                key={title}
-                className={itemClassName}
-                open={isOpen}
-                show={!openItem || isOpen}
-                onOpen={handleOpen(title)}
-                onFilterCheck={onFilterCheck}
-                priceRange={priceRange}
-              />
-            )
-          default:
-            return (
-              <AccordionFilterGroup
-                title={filter.title}
-                facets={filter.facets}
-                key={title}
-                className={itemClassName}
-                open={isOpen}
-                show={!openItem || isOpen}
-                onOpen={handleOpen(title)}
-                onFilterCheck={onFilterCheck}
+      <div className="overflow-scroll h-100 pb10">
```

Comment:
Add a CSS Handle

Reply:
I decided to remove this div to avoid possible conflicts with the client style

I just added the `overflow-scroll` and `pb9` classes [to a div that already exists](https://github.com/vtex-apps/search-result/pull/410/files#diff-d96fd9de1c6790d1afeb52b793da753dR66-R70).